src/applications/travel-pay/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function BreadCrumbs() {
+  return (
+    <va-breadcrumbs label="Breadcrumb">
+      <a href="/">Home</a>
```

Comment:
Hi @hinzed1127! This web component will start defaulting to the `uswds` version this week and there is an integration change with how it displays links. Take a look at [the code example here on Storybook](https://design.va.gov/storybook/?path=/docs/uswds-va-breadcrumbs--default). Instead of adding them to the slot, they will need to be passed into the `breadcrumb-list` prop.

Alternatively, you can set the prop `uswds="false"` on the component to delay making that update but this will need to be revisited again when this version of the web componet is removed from the component library on [May 3, 2024](https://design.va.gov/about/whats-new).

Reply:
Thanks for the heads up, @jamigibbs! Updated to that change in [480284c](https://github.com/department-of-veterans-affairs/vets-website/pull/27932/commits/480284c09217b81af85ce6effde21f381bf72d6a).